gauntlet/packages/gauntlet-serum-multisig/src/commands/setOwners.ts

```diff
@@ -2,14 +2,13 @@ import { SolanaCommand, RawTransaction, TransactionResponse } from '@chainlink/g
 import { PublicKey } from '@solana/web3.js'
 import { CONTRACT_LIST, getContract } from '@chainlink/gauntlet-solana-contracts'
 import { Result } from '@chainlink/gauntlet-core'
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
 
 export default class SetOwners extends SolanaCommand {
-  static id = 'set:owners'
+  static id = 'set_owners'
```

Comment:
As we namespace other commands we should do the same here:

```suggestion
  static id = 'multisig:set_owners'
```

This is consistent with `multisig:inspect`.